### ISSUE TITLE
Error::description is deprecated

### ISSUE THREAD

**Issue by imp** [krisprice/ipnet#20]
```
warning: use of deprecated item 'std::error::Error::description': use the Display impl or to_string()
   --> src/ipnet.rs:116:28
    |
116 |         fmt.write_str(self.description())
    |                            ^^^^^^^^^^^
    |
    = note: `#[warn(deprecated)]` on by default
```

This, as well as implementation of `Error::description` for this crate's errors.

**Comment by krisprice** [comment]
This can be removed in a future release. I tried #[allow(deprecated)] to remove the warning for now but it doesn't seem to work.

**Comment by krisprice** [comment]
Fixed in #28
